const.py

```diff
@@ -9,6 +9,7 @@
 CODE = 'code'
 MSG = 'message'
 STATUS = 'status'
+HEADERS= 'headers'
```

Comment:
This is a painful anti-pattern that we inherited from somewhere (Signal maybe?).  Putting magic values into descriptive constants is a good thing, but these aren't magic values nor does the name of the constant add any description.

Reply:
:+1: